README.md

```diff
@@ -1,24 +1,68 @@
-# radlings
-Radlings project in the scope of DPS
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
```

Comment:
Please add which node version you used so we can refer back to it if we're going to try to run it locally